website/docs/modules/ROOT/pages/depth/design-principles.adoc

```diff
@@ -367,11 +367,11 @@ reduced to simply instantiating a `trait` over and over, with tweaks.
 
 == Prior Work
 
-=== SBT
+=== sbt
 
-Mill is built as a substitute for SBT, whose problems are
-http://www.lihaoyi.com/post/SowhatswrongwithSBT.html[described here].
-Nevertheless, Mill takes on some parts of SBT (builds written in Scala, Task
+Mill is built as a substitute for sbt, whose problems are
+http://www.lihaoyi.com/post/Sowhatswrongwithsbt.html[described here].
```

Comment:
This looks like we should revert?

Reply:
True. I made a mistake here.

Reply:
I just noticed that this link is still an `http` link. Shall we make it `https`?

Reply:
It it works, yes.